contracts/rules/PricerRule.sol

```diff
@@ -70,6 +70,12 @@ contract PricerRule is Organized {
      */
     uint256 public conversionRateDecimalsFromBaseCurrencyToToken;
 
+    /**
+     * @dev Required decimals for EIP20Token. Since Pay method can be called multiple
```

Comment:
What do we mean by saying `required` here?

Reply:
It's the token decimal of EIP20Token. 
Do you think below is more clear?
'Token decimals for EIP20Token'

Reply:
Much cleaner. Required is confusing here as we are not requiring anything. Thx.